analysis_for_run.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python3
+
+"""
+Author: John Arnn
+Released: 2022-09-20
+Description:
+This script will download the reads from BSSH once run completes on sequencer, start analysis on ICA,
+and download results from ICA while sending updates to Slack at UPHL Workspace notifications channel.
+EXAMPLE:
+python analysis_for_run.py UT-VH0770-220915 -a mycosnp
+"""
+
+import os
+import sys
+import argparse
+import subprocess
+import re
+import time
+import pandas as pd
+import numpy as np
+from datetime import date
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+from importlib.machinery import SourceFileLoader
+
+config = SourceFileLoader("config","/Volumes/IDGenomics_NAS/config.py").load_module()
```

Comment:
What is this file?

I feel like we should probably move it somewhere safer, like /home/Bioinformatics

Reply:
Can we change this to `/home/Bioinformatics/config.py`? We'd have to restrict testing to the linux workstation, but it'd keep this config file "safer".